src/pages/Graph/GraphHelpTour.ts

```diff
@@ -19,7 +19,14 @@ export const GraphTourStops: { [name: string]: TourStopInfo } = {
   Graph: {
     name: 'Graph',
     description:
-      "Single click a node to see its summary and emphasize its end-to-end paths.  Double click a node to see a graph focused on that node.\nDouble click an 'external namespace' node to navigate directly to the namespace in the node's text label.",
+      "Single click a node to see its summary and emphasize its end-to-end paths. Double click a node to see a graph focused on that node.\nDouble click an 'external namespace' node to navigate directly to the namespace in the node's text label.",
+    position: PopoverPosition.auto,
+    offset: '0, -350'
+  },
+  ContextualMenu: {
+    name: 'Contextual Menu',
+    description:
+      'Right click a node to see the contextual menu for the node, such as details, traffic and inbound/outbound metrics.',
```

Comment:
You can also right-click on an edge. The description should reflect that option as well.

Reply:
@mtho11 done, also changed the weird structure of the phrase for something better.

Reply:
Just realized I forgot to add a screenshot, thanks!